tests/Tests/600__ai_apps/640__rhoam/640__rhoam.robot

```diff
@@ -5,6 +5,7 @@ Library         SeleniumLibrary
 Suite Setup     RHOAM Suite Setup
 Suite Teardown  RHOAM Suite Teardown
 
+
 *** Test Cases ***
 Verify RHOAM Is Available In RHODS Dashboard Explore Page
   [Tags]  ODS-271  Smoke  Sanity
```

Comment:
I think it should add also Tier1 until we get a proper idea of Tier1/Sanity usage